libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/dateFiltering.spec.ts

```diff
@@ -0,0 +1,80 @@
+// (C) 2023 GoodData Corporation
+
+import * as Navigation from "../../tools/navigation";
+import { DateFilter } from "../../tools/dateFilter";
+import { EditMode } from "../../tools/editMode";
+import {Kpi} from "../../tools/kpi";
+
+const dateFilter = new DateFilter();
+const editMode = new EditMode();
+
+describe("Date filtering", () => {
+    it("should disable default date correctly", { tags: "checklist_integrated_tiger" }, () => {
+        Navigation.visit("dashboard/new-dashboard");
+        dateFilter.subtitleHasValue("This month");
+    });
+
+    it("should update date filter value correctly", { tags: "checklist_integrated_tiger" }, () => {
+        Navigation.visit("dashboard/dashboard-many-rows-columns");
+        dateFilter.subtitleHasValue("All time");
+        dateFilter.openAndSelectOptionByName("This year").apply().subtitleHasValue("This year");
+    });
+
+    it("should display message on the top date filter dropdown", { tags: "checklist_integrated_tiger" }, () => {
+        Navigation.visit("dashboard/dashboard-many-rows-columns");
+        editMode.edit().isInEditMode(true);
+        dateFilter.open().isTopMessageVisibled(true);
+    });
+
+    it("should reset the unsaved selected date filter when open edit mode", { tags: "checklist_integrated_tiger" }, () => {
+        Navigation.visit("dashboard/dashboard-many-rows-columns");
+        editMode.edit().isInEditMode(true);
```

Comment:
Please help check logic of the test again. as i see it the same steps as method above

Reply:
sorry, this is my mistake when clone the test.